code/game/gamemodes/gang/gang.dm

```diff
@@ -1,7 +1,7 @@
 //gang.dm
 //Gang War Game Mode
 
-var/list/gang_name_pool = list("Clandestine", "Prima", "Zero-G", "Max", "Blasto", "Waffle", "North", "Omni", "Newton", "Cyber", "Donk", "Gene", "Gib", "Tunnel", "Diablo", "Psyke", "Osiron", "Sirius", "Sleeping Carp")
```

Comment:
@Ikarrus I thought that's what changing that line did. Sleeping Carp is no longer in the name pool so no Gang will ever be designated "Sleeping Carp" and thus it won't trigger all the other Gang Martial Arts code.